.github/workflows/coverage.yml

```
@@ -0,0 +1,67 @@
+name: Coverage
+
+on:
```

YAMLlint (1.38.0)

[warning] truthy value should be one of [false, true]

(truthy)